### PR TITLE
Doing some of the requirements for next lesson

### PR DESCRIPTION
**Pull request by dovydasjaru**
Added Named operators
To add Optional operators i will need Linas's part (For LiveRecognitionForm MessageBox.Show method, to make caption optional)

### CODE REVIEW

**Comment by dovydasjaru** [comment]
There is only one optional operator......

File: WhosThat/WhosThat/BusinessLogic/WikiData.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ public bool CheckConnection()
         {
             try
             {
-                using (client.OpenRead(ConfigurationManager.AppSettings["connectionCheck"]))
+                using (client.OpenRead(address: ConfigurationManager.AppSettings["connectionCheck"]))
```
Comment (LinasEin) [inline WhosThat/WhosThat/BusinessLogic/WikiData.cs]: Not sure if that is a good practice to use named argument type for only 1 argument.

File: WhosThat/WhosThat/View/LiveRecognitionForm.cs [hunk 2]
```diff
@@ -56,9 +56,9 @@ private void ExitBtn_Click(object sender, EventArgs e)
             this.Close();
         }
 
-        public void ShowMessage(string text, string caption)
+        public void ShowMessage(string text, string caption = "")
```
Comment (LinasEin) [inline WhosThat/WhosThat/View/LiveRecognitionForm.cs]: Maybe in this case String.Empty would be better?

**Comment by LinasEin** [comment]
I think now you should merge into master branch

**Pull request merged**